test/unit/models/test_superblocks.py

```diff
@@ -258,13 +258,24 @@ def test_superblock_size_limit(go_list_proposals):
     max_budget = 60
     prop_list = Proposal.approved_and_ranked(proposal_quorum=1, next_superblock_max_budget=max_budget)
 
-    maxgovobjdatasize = 469
+    # mock maxgovobjdatasize by setting equal to the size of a trigger
```

Comment:
Now that network is upgraded to 0.12.3.x  and we dropped 70208 compatibility code https://github.com/dashpay/dash/pull/2295 , I think we can safely drop `MAX_GOVERNANCE_OBJECT_DATA_SIZE` checks (and this mock) as discussed in https://github.com/dashpay/dash/pull/2004 . Same for dashd, see https://github.com/dashpay/dash/pull/2298.

Reply:
Yep, we can drop the entire test now, since this is testing the trigger size limit (which doesn't make sense for [reasons you mentioned earlier](https://github.com/dashpay/dash/pull/2004#issue-176258739)). I have addressed this in https://github.com/dashpay/sentinel/pull/55.